<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1406</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color: white;
background-color: #686868;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Contract_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Контракты</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Contract_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>1351</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #686868;
color: white;
}

QHeaderView::section:horizontal{
background-color: #686868;
border: 0px;
color: white;
}

QHeaderView::section:vertical{
background-color: #686868;
border: 0px;
color: white;
}

QTableWidget QTableCornerButton::section {
    background-color: #686868;
}</string>
    </property>
    <row>
     <property name="text">
      <string>Контракт 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Контракт 10</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Индекс</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Бригада</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Финансы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пиломатериалы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цемент</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кровел. изделия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Метал. изделия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Стекл. изделия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прогресс работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Заменить бригаду</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>AAA</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="team_res_tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1031</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: #686868;
}
QLabel{
color: white;
}

QTabBar::tab{
background-color: #686868;
color: white;
}

QPushButton{
background-color: #262626;
color: white;
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="res">
     <attribute name="title">
      <string>Ресурсы</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1011</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="resHlay">
       <item>
        <widget class="QTableWidget" name="Resource_widget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>505</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget{
background-color: #686868;
color: white;
}

QHeaderView::section:horizontal{
background-color: #686868;
border: 0px;
color: white;
}

QHeaderView::section:vertical{
background-color: #686868;
border: 0px;
color: white;
}

QTableWidget QTableCornerButton::section {
    background-color: #686868;
}</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>37</number>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Финансы(1000₽)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Пиломатериалы(1000kg)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Цемент(1000kg)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Кровельные изд.(1000kg)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Метал. изделия(1000kg)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Стекл. изделия(1000kg)</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Свободные ресурсы</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Требуется ресурсов</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>100000</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>1000</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="resVlayAsk">
         <item>
          <widget class="QLabel" name="Ask_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Закупить материалы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_credit">
           <property name="text">
            <string>Получить кредит</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_wood">
           <property name="text">
            <string>Заказать пиломатериалы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_cement">
           <property name="text">
            <string>Заказать цемент</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_roof">
           <property name="text">
            <string>Заказать черепицу</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_metal">
           <property name="text">
            <string>Заказать метал. изделия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ask_glass">
           <property name="text">
            <string>Заказать стекло</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="resVlaySend">
         <item>
          <widget class="QLabel" name="send_res">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Отправить материалы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_money">
           <property name="text">
            <string>Выделить средства</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_wood">
           <property name="text">
            <string>Отправить пиломатериалы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_cement">
           <property name="text">
            <string>Отправить цемент</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_roof">
           <property name="text">
            <string>Отправить черепицу</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_metal">
           <property name="text">
            <string>Отправить метал. изделия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_glass">
           <property name="text">
            <string>Отправить стекло</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="team_tab">
     <attribute name="title">
      <string>Строительные бригады</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="teamHlay">
       <item>
        <widget class="QTableWidget" name="Workers_widget">
         <property name="styleSheet">
          <string notr="true">QTableWidget{
background-color: #686868;
color: white;
}

QHeaderView::section:horizontal{
background-color: #686868;
border: 0px;
color: white;
}

QHeaderView::section:vertical{
background-color: #686868;
border: 0px;
color: white;
}

QTableWidget QTableCornerButton::section {
    background-color: #686868;
}</string>
         </property>
         <row>
          <property name="text">
           <string>Первая бригада</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Вторая бригада</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Третья бригада</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Четвертая бригада</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Пятая бригада</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Бригадир</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Укладчик</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Сантехник</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Электрик</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Стекольщик</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>684971877</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>655464129</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>616077234</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>397790836</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>668600724</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>460693868</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>696728794</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>400254209</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>815932116</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>113950884</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>503171931</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>373138214</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>953499208</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>646507861</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>508893102</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>464813395</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>440114302</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>516139329</string>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>280249938</string>
          </property>
         </item>
         <item row="3" column="4">
          <property name="text">
           <string>764735169</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>786828122</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>982863569</string>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>852225600</string>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string>756280852</string>
          </property>
         </item>
         <item row="4" column="4">
          <property name="text">
           <string>954246489</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="teamVlaySend">
         <item>
          <widget class="QLabel" name="contract_add">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Назначить подряд</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contract_add_1">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #262626;
}</string>
           </property>
           <property name="text">
            <string>Отправить бригаду</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contract_add_2">
           <property name="text">
            <string>Отправить бригаду</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contract_add_3">
           <property name="text">
            <string>Отправить бригаду</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contract_add_4">
           <property name="text">
            <string>Отправить бригаду</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contract_add_5">
           <property name="text">
            <string>Отправить бригаду</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="News">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #686868;
color: white;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Complete_contract">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>260</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #262626;
color: white;
}</string>
    </property>
    <property name="text">
     <string>Выполнить контракт</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1406</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
